baseplate/frameworks/pyramid/__init__.py

```diff
@@ -148,9 +148,10 @@ def baseplate_tween(request: Request) -> Response:
                     if (hasattr(request.matched_route, "pattern") and request.matched_route.pattern)
                     else request.matched_route.name
                 )
-            http_method = request.method.lower()
+            else:
+                http_endpoint = "404"
```

Comment:
are we sure 404 is the only reason we end up in this case?

Reply:
Pretty sure, about 85% confidence. [Docs](https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/urldispatch.html#the-matched-route)

For you to get a 5xx it has to already be in there.

My guess is that technically a middleware could mess with it and unset it or something strange, but I don't think there's benefit to supporting those strange use cases